tests/test_spelling.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""Testing the spelling of numbers
```

Comment:
@blagasz 
Sorry, I still see two files and I'm looking to the latest revision. Following @sils1297 hints should be enough. 
Looking at you repo, you can see the duplicated files (`number_spelling.py` and `spelling.py`):
![screen shot 2015-08-04 at 17 05 37](https://cloud.githubusercontent.com/assets/607262/9071375/6bd5122e-3acb-11e5-9900-6beba68c71f8.png)